docs/helm-deployment-aws_kops.md

```diff
@@ -293,7 +293,7 @@ export EFS_SERVER=<EFS_ID>.efs.<AWS-REGION>.amazonaws.com
 ```
 ### Creating a Docker registry pull secret
 
-Since we need to use protected Docker images from Quay.io, you need access to a secret with credentials to be able to pull those images.
+Since we need to use private (Enterprise-only) Docker images from Quay.io, you need access to a secret with credentials to be able to pull those images.
```

Comment:
I would rephrase this sentence in the following way:
Since we need to use private (Enterprise-only) Docker images, you need credentials to be able to pull those images from quay.io.

Reply:
Done